cmd/snap-update-ns/bootstrap.c

```diff
@@ -136,10 +160,19 @@ int partially_validate_snap_name(const char* snap_name)
     }
 }
 
+static void neuter_environment()
```

Comment:
That term feels unusual to me, maybe just `clearenv_all()` or something?

Reply:
yes, the typical abstract term would be neutralize,

neuter meaning 1 is from the wrong domain:

https://en.oxforddictionaries.com/definition/neuter

Reply:
Thanks, I inlined this into bootstrap and removed the name altogether.